backend/qemu.pm

```diff
@@ -711,10 +715,16 @@ sub start_qemu {
                 }
             }
             else {
+                my $diskmodel;
+                if (defined $vars->{"HDDMODEL_$i"}) {
```

Comment:
Since it's a new variable, it must be documented in doc/backend_vars.asciidoc file